packages/react-core/codeConnect/icons/IconProps.ts

```diff
@@ -0,0 +1,8 @@
+import { SVGProps } from 'react';
+
+export interface IconProps extends SVGProps<SVGSVGElement> {
```

Comment:
This file code isn't used/referenced anywhere outside this file, I believe it carried forward from an earlier iteration and can be deleted.